### PR TITLE
Follow-up of #70409: internal clientset switcher for registration hook

### PR DESCRIPTION
**Pull request by yue9944882**
/kind cleanup

Follow-up of #70409 

Put ClientCARegistrationHook related changes separately in this pull.

/hold 

until #70409 

/assign @deads2k 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by yue9944882** [comment]
/sig api-machinery
/priority longterm

since it doesnt belong to any milestone

**Comment by mbohlool** [comment]
cc @yliaog

**Comment by justinsb** [comment]
(Looks like you need a gofmt though)

**Comment by yue9944882** [comment]
yuh huh. gofmt again. thx for reminding!

File: pkg/master/client_ca_hook_test.go [hunk 1]
```diff
@@ -228,18 +228,18 @@ func TestWriteClientCAs(t *testing.T) {
 	}
 }
 
-func getFinalConfiMaps(client *fake.Clientset) (map[string]*api.ConfigMap, bool) {
-	ret := map[string]*api.ConfigMap{}
+func getFinalConfiMaps(client *fake.Clientset) (map[string]*corev1.ConfigMap, bool) {
```
Comment (justinsb) [inline pkg/master/client_ca_hook_test.go]: Nit: just noticed the typo here (`ConfiMaps`) but it is pre-existing so ...

Comment (yue9944882) [reply pkg/master/client_ca_hook_test.go]: killed that 😉plz re-lgtm thx

**Comment by justinsb** [comment]
/lgtm

Looks like I'm going to have to fix those aws tests :-)

**Comment by deads2k** [comment]
/approve

serve my machine master.

**Comment by yue9944882** [comment]
long live the bot @k8s-ci-robot

**Pull request merged**